flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1201,13 +1201,19 @@ bool validInputRange(int16_t min, int16_t max, uint16_t value, uint16_t offset)
  */
 static void applyDeadband(float *value, float deadband)
 {
-	if (fabsf(*value) < deadband)
+	if (deadband < 0.00001f) return;
+
+	if (fabsf(*value) < deadband) {
 		*value = 0.0f;
-	else
-		if (*value > 0.0f)
+	} else {
+		if (*value > 0.0f) {
 			*value -= deadband;
-		else
+		} else {
 			*value += deadband;
+		}
+
+		*value /= 1 - deadband;
```

Comment:
Yes, ```<1```, sorry! 

You raise excellent points. I agree that the strict range doesn't guarantee safety, although it guarantees numerical stability. I like to think we're pretty good about catching divide by 0 errors. It's such an important problem that there's [a hardware register for it](https://github.com/TauLabs/TauLabs/pull/488#discussion_r4101533).

If it's easiest, we could kick this can down the road and open an issue to decide what is the best value.